docs/api-docs/management-apis/email-templates/overview.md

```diff
@@ -0,0 +1,133 @@
+# Email Templates Overview
+
+<div class="otp" id="no-index">
+
+### On This Page
+
+- [Editing](#editing)
+- [API](#api)
+- [Overrides](#overrides)
+- [Resources](#resources)
+
+</div>
+
+This article provides and overview of BigCommerce's transactional email templates and API.
+
+## Editing
+
+To edit transactional email templates in a store's control panel, [log in](https://support.bigcommerce.com/s/article/Logging-Into-Your-Store) and navigate to **Marketing** › [**Transactional Emails**](http://login.bigcommerce.com/deep-links/manage/transactional-emails).

```

Comment:
That link gave me a 404, but the link below pulled up the Email Templates menu for me.

```suggestion
To edit transactional email templates in a store's control panel, [log in](https://support.bigcommerce.com/s/article/Logging-Into-Your-Store) and navigate to **Marketing** › [**Email Templates**](http://login.bigcommerce.com/deep-links/manage/marketing/email-templates).

```

Reply:
Its correct. You need to have the experiment turned on.